src/main/webapp/script.js

```diff
@@ -38,3 +38,33 @@ function createCommentElement(comment) {
 function addParagraph(content) {
   return "<p>" + content + "</p>";
 }
+
+function newGroupchat() {
```

Comment:
GroupChat >:(
jk do what u want hehe just suggestion
idk u said before you'd change to just "Group" or smth but since group is its own thing i guess not
would be nice to also change Groupchat --> GroupChat 4 the class

Reply:
I prefer Groupchat, since it is just one word.

Reply:
ITS 2 WORDS I SWEAR